embeddings/intro-textemb-vectorsearch.ipynb

```diff
@@ -667,7 +667,7 @@
         "\n",
         "![](https://storage.googleapis.com/github-repo/img/embeddings/textemb-vs-notebook/8.png)\n",
         "\n",
-        "Which metric should we use? Usually it depends on how each model is trained. In case of the model `textembedding-gecko@001`, we need to use inner product (dot product).\n",
+        "Which metric should we use? Usually it depends on how each model is trained. In case of the model `textembedding-gecko@003`, we need to use inner product (dot product).\n",
```

Comment:
```suggestion
        "Which metric should we use? Usually it depends on how each model is trained. In case of the model `text-embedding-004`, we need to use inner product (dot product).\n",
```